### PR TITLE
Minor changes to readme

### PR DESCRIPTION
**Pull request by sb-travelperk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,17 +47,19 @@ to configure, but up to you) to build a CI/CD over the service you have deployed
 
 ## Important Notes
 
-  * We recommend to try to run the application dockerized in your computer
+  * We recommend you to try to run the application dockerized locally on your computer
```
Comment (devvesa) [inline README.md]: Nope, `recommend you` is incorrect. It means you are the recommended one. Latin-like confusion.

File: README.md [hunk 2]
```diff
@@ -42,17 +47,19 @@ to configure, but up to you) to build a CI/CD over the service you have deployed
...
-  * We are still calibrating the assessment, and probably you won't have time
-    to finish it. This is fine. If it takes more than 4 hours leave the work as
+  * We are still calibrating the assessment, and you may not have time
+    to finish everything in the allocated time. This is fine. If it takes more than 4 hours you can leave the work as
     it is and write down what your next steps would be. If you want to spend
     two days or a week, up to you. But we don't want you to waste more of you
     time.
-  * This is not a speed test. Please don't worry if you don't finish it.
-  * Many thanks for your time and good luck!
+  * This is not a speed test. Some people will have more experience and tooling ready, some won't. Please don't worry if
```
Comment (devvesa) [inline README.md]: I think this point is over complicated, I liked more before.

File: README.md [hunk 3]
```diff
@@ -17,7 +19,9 @@ In a virtualenv, run:
     python setup.py install
     FLASK_APP=hello flask run
 
-## Tasks
+## Tasks to complete
+
+Now you have the app running, it's your turn to work :)
```
Comment (devvesa) [inline README.md]: I'd prefer not to rely on smiles. Sounds less serious and in this case, I want to give the impresion of a serious company.

**Review by devvesa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sb-travelperk) [reply README.md]: I partially disagree... but that's perfectly fine, I'm not a native English speaker :) 

But I'm pretty sure you can say:   `I recommend you (somthing/verb)`  -- without the **to** though. Let's have  aworkshop one day about that :rofl:

File: README.md (hunk 3 above)
Comment (sb-travelperk) [reply README.md]: Trust me, asking people to Terraform a Fargate cluster and set a CD pipeline sounds serious. 
But I agree, it came without me really noticing.

**Pull request merged**